### PR TITLE
[#80] exclude files from .gitignore

### PR DESCRIPTION
**Pull request by mbao01**
## Description

Exclude files in .gitignore from pdd when `--skip-gitignore` is set

## Types of changes

- [ ] Documentation Update 📝
- [ ] Build/Workflow Update 👷
- [x] Updates/Feature
- [ ] Other (please describe)

## Checklist

- [x] Lint and tests pass locally with my changes
- [ ] Added documentation (if appropriate)

### CODE REVIEW

File: bin/pdd [hunk 1]
```diff
@@ -77,8 +77,13 @@ https://github.com/cqfn/pdd/blob/master/README.md"
...
 
-  if opts['skip-gitignore']
-    raise 'For --skip-gitignore see https://github.com/cqfn/pdd/issues/80'
+  if opts['skip-gitignore'] && File.exist?('.gitignore')
+    cfg = File.new('.gitignore')
+    body = File.read(cfg)
+    extra = body.split(/\s+/).map(&:strip)
+    opts['skip-gitignore'] = extra
+    puts "Found #{body.split(/\n/).length} lines in #{File.absolute_path(cfg)}"
+    # raise 'For --skip-gitignore see https://github.com/cqfn/pdd/issues/80'
```
Comment (yegor256) [inline bin/pdd:86]: @mbao01 forgot to remove this line?

**Comment by yegor256** [comment]
@mbao01 good work! one comment from me and we are good to go

File: bin/pdd (hunk 1 above)
Comment (mbao01) [reply bin/pdd]: Removed, thank you @yegor256

**Comment by mbao01** [comment]
@yegor256  please review

**Comment by mbao01** [comment]
@yegor256 please review

**Comment by AntistasII** [comment]
@yegor256 review this PR please

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@mbao01 don't forget to run `bundle exec rake` before sending a PR

**Comment by AntistasII** [comment]
@mbao01 i think there is a problem with branch

**Comment by mbao01** [comment]
@yegor256 please review when you have some time

**Comment by yegor256** [comment]
@rultor merge

**Comment by AntistasII** [comment]
@yegor256 do have some server problems here? Looks like some folders are missing.. We have same mistake in another PR

**Pull request merged**